Sylvanian clans / lords / settlement descriptions / townspeople

hunharibo opened the pull request:
Also changed 1 hat variation and added bandits/merchants/Townsfolk that still probably need to be changed though for now..

tristanka commented on Abilities/AbilityComponent.cs:
```diff
@@ -48,7 +48,7 @@ public AbilityComponent(Agent agent) : base(agent)
         public void SelectAbility(int index)
         {
-            if (_knownAbilities.Count > 0)
+            if (_knownAbilities.Count > 0 && index >= 0)
```
If index was somehow less than 0, the index could end up negative as well, so this ensures we avoid that.

tristanka commented on Abilities/FireBallAbilityScript.cs:
```diff
@@ -96,15 +96,15 @@ protected override void OnPhysicsCollision(ref PhysicsContact contact)
                 MatrixFrame frame = MatrixFrame.Identity;
                 var psys = ParticleSystem.CreateParticleSystemAttachedToEntity("psys_burning_projectile_default_coll", explosion, ref frame);
-                var globalFrame = new MatrixFrame(Mat3.CreateMat3WithForward(in collisionnormal), collisionpoint);
+                var globalFrame = new MatrixFrame(Mat3.CreateMat3WithForward(in collisionNormal), collisionPoint);
```
In this file, I just camel cased some variables.

Z3rca reviewed (approved):
fairly understandable also for me :)

hunharibo reviewed (approved):
Nice work, approved!

Kysaro committed:
Sylvanian clans / lords / settlement descriptions / townspeople

Also changed 1 hat variation and added bandits/merchants/Townsfolk that still probably need to be changed though for now..

hunharibo committed:
Delete TOW_log2021-06-26.txt
